XamarinCommunityToolkit/Behaviors/BaseBehavior.shared.cs

```diff
@@ -37,6 +37,12 @@ protected override void OnAttachedTo(TView bindable)
 				};
 				SetBinding(BindingContextProperty, defaultBindingContextBinding);
 			}
+
+			OnAttached();
+		}
+
+		internal virtual void OnAttached()
+		{
 		}
```

Comment:
This method must contain all code that the currently **protected** method contains.
The protected method will be pretty slim

```csharp
protected override void OnAttachedTo(TView bindable)
{
	base.OnAttachedTo(bindable);
	OnAttached(bindable)
}
```